lib/doc-links.js

```diff
@@ -122,6 +122,14 @@ module.exports = function fixLinks (dir, version, callback) {
     oldLinks.forEach(function (oldLink, i) {
       content = content.replace(oldLink, newLinks[i])
     })
+
+    // Work around kramdown's misinterpretation of pipe as a table header
+    // https://github.com/electron/electron.atom.io/issues/556
+    content = content
+      .split('\n')
+      .map(line => line.match(/^\* `/) ? line.replace(/ \| /g, ' &#124; ') : line)
```

Comment:
Adding a `\s*` between the `^` and `\*` will replace pipes in sublists as well, not sure if we have any now but it seems possible that we could in the near future.